packages/api/src/mapping/PropertyValueMapping.ts

```diff
@@ -71,3 +78,9 @@ export interface PropertyValueClientToWire {
   sensorTimeseries: TimeSeriesProperty<string | number>;
   geotimeSeriesReference: GeotimeSeriesProperty<GeoJSON.Point>;
 }
+export type getWirePropertyValueFromClient<
```

Comment:
I find this type being lower case to be a bit confusing

Reply:
Ah no I agree, bad convention on my part